compliance/google/long_stopped_instances/google_long_stopped_instances.pt

```diff
@@ -197,10 +228,10 @@ policy "pol_utilization" do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} google cloud instances in Stopped State for Over {{parameters.param_stoped_days}} Day(s)"
     detail_template <<-EOS
 # Google Instance Out of Compliance
-|Instance ID| Name | Status | Zone | Tags |
-|:---------:|-----:|-------:|-----:|-----:|
+| Project ID | Instance ID | Name | Status | Zone | Tags |
```

Comment:
please sort by project id and region/zone.  rename zone to regions and mov next to projectid.

Reply:
I have addressed this comment please review the changes.